### PR TITLE
Add layer field to event

### PR DESCRIPTION
**Pull request by fgksgf**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Related #8825.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/changelog/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please fix the CI.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 * Fix OAP Self-Observability metric `GC Time` calculation.
 * Set `SW_QUERY_MAX_QUERY_COMPLEXITY` default value to `1000`
 * Webapp module (for UI) enabled compression.
+* [Breaking Change] Add layer field to event, report an event without layer will not be allowed.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:22]:
```suggestion
* [Breaking Change] Add layer field to event, report an event without layer is be allowed.
```
 Future tense in the change log is strange.

Comment (fgksgf) [reply docs/en/changes/changes.md:22]: make sense to me

File: test/e2e-v2/script/prepare/setup-e2e-shell/install-swctl.sh [hunk 2]
```diff
@@ -22,9 +22,18 @@
...
   curl -kLo skywalking-cli.tar.gz https://github.com/apache/skywalking-cli/archive/${SW_CTL_COMMIT}.tar.gz
   tar -zxf skywalking-cli.tar.gz --strip=1
-  make install DESTDIR=$BIN_DIR
+  VERSION=${SW_CTL_COMMIT} make install DESTDIR=$BIN_DIR
+}
+
+if ! command -v swctl &> /dev/null; then
+  echo "swctl is not installed"
+  install_swctl
+elif ! echo "$(swctl --version)" | grep -q "${SW_CTL_COMMIT}"; then
```
[the remark this answers is not in the document]
Comment (fgksgf) [reply test/e2e-v2/script/prepare/setup-e2e-shell/install-swctl.sh:35]: ignore

[the remark this answers is not in the document]
Comment (fgksgf) [reply test/e2e-v2/script/prepare/setup-e2e-shell/install-swctl.sh:35]: ignore

File: oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/rest/EventRestServiceHandler.java [hunk 3]
```diff
@@ -41,27 +44,35 @@ public class EventRestServiceHandler {
...
-            new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
+                "event_in_latency", "The process latency of event data",
+                new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
         );
         errorCounter = metricsCreator.createCounter(
-            "event_error_count", "The error number of event analysis",
-            new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
+                "event_error_count", "The error number of event analysis",
+                new MetricsTag.Keys("protocol"), new MetricsTag.Values("http")
         );
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/rest/EventRestServiceHandler.java:61]: Consider reverting these re-formats?

Comment (fgksgf) [reply oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/rest/EventRestServiceHandler.java:61]: Ok, it seems IDEA formats it automatically.

Comment (wu-sheng) [reply oap-server/server-receiver-plugin/skywalking-event-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/event/rest/EventRestServiceHandler.java:61]: I think only `second format` would change this, at least my IDEA acts in this way.

**Review by wu-sheng — APPROVED** [review]
I am good once e2e passed.

**Comment by wu-sheng** [comment]
@fgksgf This PR brings some issues to the codebase. @wankai123 fixed them through https://github.com/apache/skywalking/pull/11413

**Pull request merged**